deploy/charts/cert-manager/templates/webhook-mutating-webhook.yaml

```diff
@@ -33,21 +27,15 @@ webhooks:
           - UPDATE
         resources:
           - "*/*"
-    {{- if $isV1AdmissionRegistration }}
     admissionReviewVersions: ["v1", "v1beta1"]
     timeoutSeconds: {{ .Values.webhook.timeoutSeconds }}
-    {{- end }}
```

Comment:
I don't really understand the original hack: https://github.com/jetstack/cert-manager/pull/3167/files but that PR contained a bunch of other changes, to various bazel files. Should we revert those changes too?